samples/servicebus-queue/deploy/externalmetric.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: azure.com/v1alpha1
+kind: ExternalMetric
+metadata:
+  name: queuemessages
```

Comment:
Can't recall but do we deploy these per namespaces or not? If yes, it might be good to emphasize this here

Reply:
Yes the do get deployed per namespace.  Added a note in the Read me.